src/fillTank.test.js

```diff
@@ -1,11 +1,100 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should be declared ', () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('should calculate full tank If the amount is not given', () => {
+    const customer = {
+      money: 3000, // customer account balance
+      vehicle: {
+        maxTankCapacity: 40, // fuel tank volume
+        fuelRemains: 0, // Remaining fuel in the tank
+      },
+    };
+
+    fillTank(customer, 30);
 
+    expect(customer.money).toBe(1800);
+    expect(customer.vehicle.fuelRemains).toBe(40);
```

Comment:
The test description mentions calculating a full tank if the amount is not given, but the test case provides an amount of 30. Ensure the test case aligns with the description, or update the description if the test case is correct.